backend/routes/rides.js

```diff
@@ -87,16 +101,20 @@ router.post('/:userID', async function (req, res, next) {
     const collection = client.dbCollection(collectionName);
     collection.insertOne(rideDocument, function (err, record) {
         if (err) {//insert a record with an existing _id value
-            console.log("New Ride Error");
+            if (DEBUG) {
+                console.log("New Ride Error");
+            }
```

Comment:
Can you also get rid of this change here or keep it locally?

Reply:
Ah yes I missed this, I'll get rid of it